package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "ghost-s3-storage",
-  "version": "0.2.2",
-  "description": "Store Ghost's media at AWS S3 storage. Modified version of ghost-s3 (https://www.npmjs.org/package/ghost-s3) and ghost-aws-s3 (https://github.com/kimar/ghost-aws-s3) ",
+  "name": "ghost-s3-compat",
+  "version": "0.1.0",
```

Comment:
Per the first two points in http://semver.org/#faq, I think 1.0.0 might be appropriate. But you can make the final call.